src/python/grpcio_reflection/grpc_reflection/v1alpha/proto_reflection_descriptor_database.py

```diff
@@ -21,7 +21,9 @@
 from typing import Any, Dict, Iterable, List, Set
 
 from google.protobuf.descriptor_database import DescriptorDatabase
-from google.protobuf.descriptor_pb2 import FileDescriptorProto
+from google.protobuf.descriptor_pb2 import (
+    FileDescriptorProto,
+)  # pytype: disable=pyi-error
```

Comment:
Can you explain the issue that needs to be silenced here?

Reply:
didn't need to be silenced anymore, removed the `# pytype: disable=pyi-error` and verified on local.